feelpp/feel/feeldiscr/timeset.hpp

```diff
@@ -707,7 +722,8 @@ class TimeSet
                 if ( func.worldComm().isActive() )
                 {
                     // interpolate field on visualisation space
-                    interpolate( M_scalar_p1, func, M_nodal_scalar[__fname] );
+                    interpolate( M_scalar_p1, func, M_nodal_scalar[__fname], INTERPOLATE_DIFFERENT_MESH );
```

Comment:
why there is INTERPOLATE_DIFFERENT_MESH ?

Reply:
it is not used anymore, it should be removed